en/flight_controller/nxp_rddrone_fmuk66.md

```diff
@@ -0,0 +1,119 @@
+  - Gyro: FXAS21002CQ
+  - Magnetometer: BMM150
+  - Barometer ML3115A2
+  - Barometer BMP280
+- **GPS:**
+  - ublox Neo-M8N GPS/GLONASS receiver; integrated magnetometer IST8310
+
+
+This FMU is provided only as a kit, and includes Segger Jlink EDU mini debugger, DCD-LZ debugger adapter, USB-TTL-3V3 console cable, HolyBro GPS module, battery power module, SDCard and case, screws and stickers.
+Telemetry radios (HGD-TELEM433 and HGD-TELEM915) must be purchased separately to match ISM band frequencies used in your country.
+
+![RDDRONE-FMUK66 FMU Kit](../../assets/flight_controller/nxp_rddrone_fmuk66/RDDRONE-FMUK66-KIT-IMG.webp)
+
+A "Lite" version RDDRONE-FMUK66L is also available which does not include the power module, GPS, Jlink or USB-TTL-3V3 console cable or SDCard.
+
+Additional information can be found in the [Technical Data Sheet](https://www.nxp.com/design/designs/px4-robotic-drone-fmu-rddrone-fmuk66:RDDRONE-FMUK66). <!-- www.nxp.com/rddrone-fmuk66 -->
+
+
+## Purchase
+
+**RDDRONE-FMUK66** reference design kit may be purchased direct from NXP or from any of NXP's authorised worldwide network of [electronics distributors](https://www.nxp.com/support/sample-and-buy/distributor-network:DISTRIBUTORS).
+
+- [Purchase Link](https://www.nxp.com/design/designs/px4-robotic-drone-fmu-rddrone-fmuk66:RDDRONE-FMUK66#buy) (www.nxp.com)
+- Telemetry radios are purchased separately depending on frequency band:
+  - [HGD-TELEM433](https://www.nxp.com/part/HGD-TELEM433)
+  - [HGD-TELEM915](https://www.nxp.com/part/HGD-TELEM915)
+
+
+> **Note** *RDDRONE-FMUK66* FMU is also included in the complete HoverGames drone kit: [KIT-HGDRONEK66](https://www.nxp.com/applications/solutions/industrial/aerospace-and-mobile-robotics/uavs-drones-and-rovers/nxp-hovergames-drone-kit-including-rddrone-fmuk66-and-peripherals:KIT-HGDRONEK66#buy)
+
+
+
+## Connectors
+
+[Connector Diagram]
+
+## Pinouts
+
+[Pinouts listing or link]
+
+## Dimensions
+
+[Dimensions]
+
+
+
+## Assembly/Setup
+
+https://nxp.gitbook.io/hovergames
+
+## Building Firmware
+
+> **Tip** Most users will not need to build this firmware!
+  It is pre-built and automatically installed by *QGroundControl* when appropriate hardware is connected.
+
+To [build PX4](https://dev.px4.io/master/en/setup/building_px4.html) for this target:
+```
+make XXXXXX
+```
+
+
+## Debug Port
+
+The [PX4 System Console](http://dev.px4.io/master/en/debug/system_console.html) and the [SWD interface](http://dev.px4.io/master/en/debug/swd_debug.html) run on the [DCD-LZ FMU Debug](https://nxp.gitbook.io/hovergames/rddrone-fmuk66/connectors/debug-interface-dcd-lz) port.
+
+NXP's DCD-LZ is a 7 pin JST-GH connector and adds the nRST/MCU_RESET pin to the [Pixhawk 6-Pin standard debug port](https://pixhawk.org/pixhawk-connector-standard/#dronecode_debug).
+
+The DCD-LZ breakout adapter permits the use of a standard 10 pin JTAG/SWD interface (i.e. using the Segger Jlink) and a standard 5 pin FTDI USB-TTL-3V3 type cable.
```

Comment:
1. Maybe "The DCD-LZ breakout adapter" to "The DCD-LZ breakout adapter (included in the kit)"
2. I'm sure I saw some docs with an image of this adapter connected somewhere in your docs. Would be great to show that image and link.

Reply:
@igalloway Do you know of an image in your docs showing the adaptor setup? I.e. answer to point 2 ^^^^